src/translations.js

```diff
@@ -293,6 +303,7 @@ const wakatimeCardLocales = {
     sk: "Žiadna kódovacia aktivita tento týždeň",
     tr: "Bu hafta herhangi bir kod yazma aktivitesi olmadı",
     pl: "Brak aktywności w tym tygodniu",
+    uz: "Bu hafta faol bo'lmadi",
```

Comment:
@MrMuminov I don't speak uzbek but is there a reason you chose `Bu hafta faol bo'lmadi` over `
Bu hafta kodlash faoliyati yo'q` that Google translate suggests. I guess because it is shorter?

Reply:
I translated it here as a general activity.

If only coding is meant here, it would be correct to say `Bu hafta kod yozish faoliyati yo'q`

Reply:
Great. Let's leave it as it is then.